Interni/NormeProgetto/components/norme-c4.tex

```diff
@@ -0,0 +1,162 @@
+Tali incontri devono attenersi al regolamento seguente:
+\begin{itemize}
+\item La partecipazione a questi incontri è limitata ai membri del gruppo;
+\item Prima di ogni incontro deve essere redatto l'ordine del giorno dal \RdP{};
+\item Prima di ogni incontro deve essere comunicata la data e l'ora della riunione sull'apposito canale \glo{Slack} del gruppo;
+\item A inizio riunione devono essere nominati due dei partecipanti per redigere il verbale.
+\end{itemize}
+I partecipanti sono tenuti a informare per tempo della loro assenza nel caso non potessero partecipare all'incontro.
+\paragraph{Organizzazione della comunicazione tra i membri del gruppo}
+Il canale di comunicazione principale è un \glo{workspace} \glo{Slack}.
+Il workspace è suddiviso nei seguenti canali accessibili a tutti:
+\begin{itemize}
+\item \textbf{annunci}: canale dedicato alla comunicazione dell'ordine del giorno delle riunioni e ad annunci di interesse generale;
+\item \textbf{chiamate}: canale dedicato alla comunicazione dei link alle chiamate zoom, siano esse ricorrenti o meno;
+\item \textbf{date}: canale dedicato alla comunicazione di scadenze e date di interesse per il gruppo come ad esempio seminari o riunioni esterne;
+\item \textbf{documenti-vari}: canale dedicato alla condivisione e archiviazione di bozze non finalizzate di documenti redatti dal gruppo;
+\item \textbf{general}: canale dedicato a comunicazioni senza un topic specifico tra i membri del gruppo.
+\end{itemize}
+Oltre a questi canali vi sono canali accessibili solo a membri del gruppo che stanno svolgendo ruoli specifici che vengono creati e resi accessibili all'occorrenza.
+\subsubsection{Comunicazioni esterne}
+\paragraph{Descrizione}
+In questa sezione vengono descritte le modalità e le norme adottate per organizzare in modo efficace la comunicazione con soggetti esterni.
+Vengono in particolare esposte le modalità e le norme inerenti a:
+\begin{itemize}
+\item Modalità di comunicazione con soggetti esterni;
+\item Incontri formali esterni.
+\end{itemize}
+\paragraph{Modalità di comunicazione con soggetti esterni}
+Tutte le comunicazioni con soggetti esterni devono avvenire tramite l'indirizzo di posta elettronica \url{\Mail}. Nel caso non rientri nei compiti del proprio ruolo, prima di inviare un'email utilizzando l'indirizzo del gruppo è obbligatorio consultare il \RdP{}.
+\paragraph{Incontri formali esterni}
+Gli incontri formali esterni vengono svolti tramite \glo{Zoom}. Come negli incontri formali interni vengono nominati due dei partecipanti appartenenti al gruppo per redigere il verbale dell'incontro. 
+
+\subsubsection{Lavoro a distanza}
+Il gruppo ha sviluppato le precedenti norme con l'obbiettivo di diminuire il più possibile l'impatto che le restrizioni causate dal virus \textit{SARS-CoV2} comportano sul lavoro. Il gruppo ha favorito soluzioni di lavoro da remoto e piattaforme di teleconferenza rispetto a meeting in persona.
```

Comment:
forse superfluo, lo avevo aggiunto perchè mi sembrava importante specificare la ragione delle nostre scelte

Reply:
"dei due partecipanti"

Reply:
Va bene perchè permette una futura espansione in caso di fine restrizioni del covid